hal/x86_64_efi.c

```diff
@@ -0,0 +1,245 @@
+/* skeleton.c
+ *
+ * Stubs for custom HAL implementation. Defines the 
+ * functions used by wolfboot for a specific target.
+ *
+ * Copyright (C) 2021 wolfSSL Inc.
+ *
+ * This file is part of wolfBoot.
+ *
+ * wolfBoot is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfBoot is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
+ */
+
+#include <stdint.h>
+#include <target.h>
+
+#include "image.h"
+#include "loader.h"
+
+#include <efi/efi.h>
+#include <efi/efilib.h>
+
+#ifdef __WOLFBOOT
+void hal_init(void)
+{
+}
+
+void hal_prepare_boot(void)
+{
+        Print(L"hal_prepare_boot\n");
```

Comment:
Check indentation throughout... should be 4 spaces.

Reply:
it should be ok now